### PR TITLE
Compose code for App Distribution

### PR DESCRIPTION
**Pull request by b1no0019**
- Added Jetpack Compose option in `EntryChoiceActivity`
- Added `ComposeMainActivity`
- Added AppDistribution for business logic
- Added `AppDistributionViewModel`
- Added `ComposeMainActivity` in `AndroidManifest`

### CODE REVIEW

**Comment by thatfiredev** [comment]
@b1no0019 thanks for fixing the CLA :)
I'll find some time to review this PR today

File: appdistribution/app/src/main/java/com/google/firebase/appdistributionquickstart/ui/theme/Theme.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /* Other default colors to override
+    background = Color(0xFFFFFBFE),
+    surface = Color(0xFFFFFBFE),
+    onPrimary = Color.White,
+    onSecondary = Color.White,
+    onTertiary = Color.White,
+    onBackground = Color(0xFF1C1B1F),
+    onSurface = Color(0xFF1C1B1F),
+    */
```
Comment (thatfiredev) [inline appdistribution/app/src/main/java/com/google/firebase/appdistributionquickstart/ui/theme/Theme.kt:29]: I understand this comment was added by Android Studio, but I think we can remove it.

```suggestion
```

File: appdistribution/app/src/main/java/com/google/firebase/appdistributionquickstart/kotlin/ComposeMainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }, content = { it ->
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally, modifier = Modifier
+                .fillMaxSize()
+                .padding(it)
+        ) {
+            Spacer(modifier = Modifier.height(30.dp))
+            Image(
+                painterResource(R.drawable.firebase_lockup_400),
+                contentDescription = "",
```
Comment (thatfiredev) [inline appdistribution/app/src/main/java/com/google/firebase/appdistributionquickstart/kotlin/ComposeMainActivity.kt:90]: Can we add a content description to this image?
```suggestion
                contentDescription = "Firebase logo",
```

**Review by thatfiredev — COMMENTED** [review]
This looks great! I just have a few suggestions

**Comment by b1no0019** [comment]
Thanks for the review @thatfiredev. All suggestions were applied.

**Review by thatfiredev — APPROVED** [review]
LGTM

**Pull request merged**